.gitignore

```diff
@@ -1,2 +1,3 @@
 .flatpak-builder/
 build/
+repo/
```

Comment:
Why is this filter required?

Reply:
Oh oops, that was a folder being created by flatpak builder. I will remove that and maybe submit a separate mini PR for it.